app/components/task/extension-form.js

```diff
@@ -0,0 +1,124 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { resource, use } from 'ember-resources';
+import { TrackedMap } from 'tracked-maps-and-sets';
+import ENV from 'website-my/config/environment';
+import { action } from '@ember/object';
+import { WARNING_INVALID_NEW_ETA } from '../../constants/user-status';
+import { toastNotificationTimeoutOptions } from '../../constants/toast-notification';
+import { inject as service } from '@ember/service';
+
+export default class ExtensionFormComponent extends Component {
+  @tracked createExtensionRequest = false;
+  @service toast;
+  @service userState;
+
+  oldETA = new Date(this.args.task.endsOn * 1000).toLocaleString();
+
+  @use load = resource(({ on }) => {
+    const state = new TrackedMap();
+    const controller = new AbortController();
+
+    on.cleanup(() => controller.abort());
+    (async () => {
+      if (this.args.task) {
+        state.set('isLoading', true);
+        try {
+          const response = await fetch(
+            `${ENV.BASE_API_URL}/extensionRequests/self/?taskId=${this.args.task.id}`,
```

Comment:
Can we not edit the request at this point in time?  Are there any plans for the same ?
![image](https://user-images.githubusercontent.com/78433013/213881015-30c9acb8-f9bc-4e4a-9742-97354278880f.png)

Reply:
No as a user you can't edit an extension request but yes in the future we may consider doing that.